pkg/cluster/describe.go

```diff
@@ -103,6 +103,16 @@ func PrintClusterDesctipion(connection *sdk.Connection, cluster *cmv1.Cluster) e
 		shard = shardPath.Body().HiveConfig().Server()
 	}
 
+	var computesStr string
+	if cluster.Nodes().AutoscaleCompute() != nil {
+		computesStr = fmt.Sprintf("%d-%d (Autoscaled)",
```

Comment:
Wonder if theres a better way to present this; `Min: x, Max: y`.

Not entirely sure

Reply:
I wrote like this because it aligns with `rosa describe cluster` and `list macinepools`